src/client/commands/submit.rs

```diff
@@ -183,6 +184,13 @@ pub async fn submit_computation(
         )
     }
 
+    if let Some(ref stdout) = stdout {
```

Comment:
This is incorrect, these environment variables need to be created by Tako, not HQ, because they can contain placeholders. @spirali right?

Reply:
Yes, if we allow to insert dynamic placeholdres (like DATE / WORKDIR) then it have to be done in launcher setup